test/unit/services/lnd/onchain-service.spec.ts

```diff
@@ -5,8 +5,6 @@ import {
   extractOutgoingTransactions,
 } from "@services/lnd/onchain-service"
 
-jest.mock("@services/redis", () => ({}))
```

Comment:
did you test this with detectOpenHandles?

Reply:
yup, it was connecting to redis.